### PR TITLE
VIITE-1255 editing ely code

### PR DESCRIPTION
**Pull request by pedrosag**

### CODE REVIEW

File: digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/ProjectServiceSpec.scala [hunk 1]
```diff
@@ -411,48 +411,50 @@ class ProjectServiceSpec extends FunSuite with Matchers with BeforeAndAfter {
...
-        Seq(), None)
-      val project = projectService.createRoadLinkProject(rap)
-      val roadWays = roadwayDAO.fetchAllByRoadAndPart(roadNumber, roadPartNumber)
-      val projectLinks = projectLinkDAO.fetchProjectLinks(project.id)
-
-      val reservedPart2 = ProjectReservedPart(0L, roadNumber, roadPartNumber, Some(1000L), Some(Continuous), Some(newReserveEly))
-
-      projectService.validateReservations(reservedPart2, Some(projectEly), projectLinks, roadWays) should be(Some(s"$ErrorFollowingPartsHaveDifferingEly $errorRoad"))
-    }
+    // ELY CODE IS NO LONGER VALIDATED
```
Comment (Campanha) [inline digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/ProjectServiceSpec.scala]: remove commented code?

File: digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/ProjectServiceSpec.scala [hunk 2]
```diff
@@ -411,48 +411,50 @@ class ProjectServiceSpec extends FunSuite with Matchers with BeforeAndAfter {
...
-        Seq(), None)
-      val project = projectService.createRoadLinkProject(rap)
-      mockForProject(project.id, roadwayAddressMapper.getRoadAddressesByRoadway(roadwayDAO.fetchAllByRoadAndPart(5, 207)).map(toProjectLink(project)))
-      projectService.saveProject(project.copy(reservedParts = Seq(reservedPart1)))
-      val roadWays = roadwayDAO.fetchAllByRoadAndPart(roadNumber, roadPartNumber)
-      val projectLinks = projectLinkDAO.fetchProjectLinks(project.id)
-      val reservedPart2 = ProjectReservedPart(0L, roadNumber, roadPartNumber, Some(1000L), Some(Continuous), Some(projectEly))
-      projectService.validateReservations(reservedPart2, Some(projectEly), projectLinks, roadWays) should be(None)
-    }
+    // ELY CODE IS NO LONGER VALIDATED
```
Comment (Campanha) [inline digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/ProjectServiceSpec.scala]: remove commented code?

File: digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/ProjectServiceSpec.scala (hunk 1 above)
Comment (pedrosag) [reply digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/ProjectServiceSpec.scala]: Removed!

File: digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/ProjectServiceSpec.scala (hunk 2 above)
Comment (pedrosag) [reply digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/ProjectServiceSpec.scala]: Removed!

**Comment by Campanha** [comment]
Renumbering should and is not allowing the update of Ely code **which is fine**.
Should ely field be disabled in this operation?

**Comment by Campanha** [comment]
W shoulde also add one more test for the https://github.com/finnishtransportagency/viite/pull/421/commits/b09dc8428ee4b4ceb55c4b42f6f2c2c7f8dc722c
where the ELY change continuity code should also prevail against the Missing end of road continuity cases.

**Comment by pedrosag** [comment]
> W shoulde also add one more test for the [b09dc84](https://github.com/finnishtransportagency/viite/commit/b09dc8428ee4b4ceb55c4b42f6f2c2c7f8dc722c)
> where the ELY change continuity code should also prevail against the Missing end of road continuity cases.

@Campanha Test created

**Comment by pedrosag** [comment]
Last commit (3170dfc5ac66eaaae5ec795107cc5d5546782e43)  deals with the Numbering operations, I forgot to change the default commit message.

**Pull request merged**